### DISCUSSION TITLE
Support for other forms (rectangles, circles, ellipses)

### DISCUSSION THREAD

**Discussion by RPDeshaies** [steveruizok/perfect-freehand#14] [Ideas]
I was wondering if you had any plans to make this library supports things like drawing "free hand looking" rectangles, circles, ellipsis, etc. in the future ?

I'm currently using [RoughJS](https://roughjs.com/) in [Fari](https://fari.app/) (and open source table-top rpg application), but I really like the look of perfect-free hand and was considering perhaps using that in the near future 😄

**Comment by steveruizok** [comment]
Hey, glad to hear you like it. I'm not planning to add other shapes to this
library. However, it could certainly be used to create such a system of
shapes!

If you're interested in trying, I've had some good results running shapes
and icons through the library. I believe you could use a seed to randomize
the easing curve in order to make things a bit more interesting, in the
same way as RoughJS.
